amqp.reflect/chart/templates/service-zilla.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: zilla
+  labels:
+    app.kubernetes.io/instance: zilla
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: zilla
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/instance: zilla
+    spec:
+      containers:
+        - name: zilla
+          command: ["/bin/sh", "-c", "/opt/zilla/zilla start -v -e"]
```

Comment:
Can we use `args` instead of `command` to rely on docker image `ENTRYPOINT` implicitly?
This will make these example k8s pod specs resilient to a change in zilla `ENTRYPOINT` location.

Suggest placing `image` and `imagePullPolicy` after `name` and before `args`.

Reply:
done